src/com/aca/LarisaGabrielyan/Homework1/Product.java

```diff
@@ -0,0 +1,24 @@
+package com.aca.LarisaGabrielyan.Homework1;
```

Comment:
read this article for the package naming conventions 

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html